examples/src/main/java/alluxio/cli/AlluxioFrameworkIntegrationTest.java

```diff
@@ -127,7 +127,7 @@ public Boolean apply(Void input) {
   private void startAlluxioFramework(Map<String, String> extraEnv) {
     String startScript = PathUtils.concatPath(Configuration.get(PropertyKey.HOME),
         "integration", "bin", "alluxio-mesos.sh");
-    ProcessBuilder pb = new ProcessBuilder(startScript, mMesosAddress, "-w");
+    ProcessBuilder pb = new ProcessBuilder(startScript, mMesosAddress);
```

Comment:
this flag was doing nothing previously - it needs to come before the other arguments to be noticed